src/matrix.h

```diff
@@ -45,13 +45,7 @@ namespace bla
             std::swap(data_, m.data_);
         }
 
-        ~Matrix() 
-        { 
-            if(DelData())
-            {
-                delete[] data_;
-            } 
-        }
+        ~Matrix() { ; }
```

Comment:
We should be consistent with empty brackets. Use {} as in the constructor instead of { ; }.